tools/webpack-plugin/src/BacktracePlugin.ts

```diff
@@ -48,30 +52,40 @@ export class BacktracePlugin implements WebpackPluginInstance {
             for (const [asset, sourcePath, sourceMapPath] of entries) {
                 logger.time(`[${asset}] process source and sourcemap`);
 
+                const stats: AssetStats = {};
+                assetStats.set(asset, stats);
+
                 let debugId: string;
                 try {
                     debugId = await this._sourceMapProcessor.processSourceAndSourceMapFiles(sourcePath, sourceMapPath);
+                    stats.debugId = debugId;
+                    stats.processSource = true;
                     logger.timeEnd(`[${asset}] process source and sourcemap`);
                 } catch (err) {
                     logger.timeEnd(`[${asset}] process source and sourcemap`);
                     logger.error(`[${asset}] process source and sourcemap failed:`, err);
+                    stats.processSource = err instanceof Error ? err : new Error('Unknown error.');
                     continue;
                 }
 
                 if (this._sourceMapUploader) {
                     logger.time(`[${asset}] upload sourcemap`);
                     try {
-                        await this._sourceMapUploader.upload(sourceMapPath, debugId);
+                        const result = await this._sourceMapUploader.upload(sourceMapPath, debugId);
```

Comment:
do we upload source map one by one instead of in an archive?

Reply:
Right now, one by one. This is out of scope of this PR.

Reply:
Yeah sure, this is definitely something that doesn't belong to this pull request, however, I saw the code and I asked a question ^^. Thank you!